fallback_builtins.h

```diff
@@ -71,6 +71,24 @@ static inline __m512i _mm512_zextsi128_si512(__m128i a) {
 
 #endif // __AVX2__
 
+/* Missing zero-extension AVX and AVX512 intrinsics.
+ * Fixed in version 15.7
```

Comment:
This should specify version 15.7 of what exactly, not that it is impossible to understand without it specified, it just makes it easier to understand when doing a quick read of the comment.

Reply:
@Dead2 
Comment updated.

Reply:
Excellent 👍